extensions/layout/src/org/sbml/jsbml/ext/layout/BoundingBox.java

```diff
@@ -364,7 +381,7 @@ public void setPosition(Point point) {
       logger.warn(MessageFormat.format(
         ResourceManager.getBundle("org.sbml.jsbml.resources.cfg.Messages").getString("UNDEFINED_ATTRIBUTE"),
         "name", getLevel(), getVersion(), getElementName()));
-      // TODO: This must be generally solved. Here we have an SBase with ID but without name!
+      // TODO 2013/12: This must be generally solved. Here we have an SBase with ID but without name!
```

Comment:
Not sure why 2013/12 was added here. But in general, this is probably something that need to be reviewed as now in L3V2 everything can have id and name. And may be the layout package changed in the final specification as well.